pipeline/Makefile

```diff
@@ -18,4 +18,5 @@ TF_PATH = $(STACK_ROOT)/terraform
 
 $(val $(call stack_setup))
 
-include $(ROOT)/$(STACK_ROOT)/transformer/Makefile
+include $(ROOT)/pipeline/transformer/Makefile
```

Comment:
fyi; necessary to hardcode what was previously `$(STACK_ROOT)` here because the `include` overwrites it. There are better ways to do this (eg with secondary expansion) but I think that's out of scope.